understanding/20/contrast-enhanced.html

```diff
@@ -126,7 +126,19 @@ <h2>Intent of Contrast (Enhanced)</h2>
          in charts, graphs, diagrams, and other non-text-based information, which is covered by
          <a href="non-text-contrast">Success Criterion 1.4.11 Non-Text Contrast</a>.         
       </p>
-      
+
+      <div class="note">
+         <p>Text used as part of a logo or logotype is exempted from contrast requirements,
+            under the assumption that logos/logotypes must comply with stricter color choices mandated
+            by corporate identity guidelines.
+            However, this can be problematic when the logo or logotype also acts as a
+            <em>user interface component</em> (such as a link or other interactive control).
+            In these cases, authors should choose a variant of the logo or logotype that has sufficient
+            contrast, if allowed by the corporate identity guidelines.
+            Alternatively, authors should provide an equivalent <em>user interface component</em>
+            which serves the same purpose and does meet the contrast requirements.</p>
```

Comment:
```suggestion
         <p>Text used as part of a logo or logotype is exempted from contrast requirements, under the assumption that logos/logotypes must comply with stricter color choices mandated by corporate identity guidelines. However, this can be problematic when the logo or logotype also acts as a <em>user interface component</em> (such as a link or other interactive control). Some part of the control must meet the 3:1 contrast ratio so users can identify that there is a control there. Authors could choose a variant of the logo or logotype that has sufficient contrast, include a border with contrast, or provide an equivalent <em>user interface component</em> which serves the same purpose and does meet the contrast requirements.</p>
```

Reply:
A suggestion to avoid "should".